pkg/mcs/scheduling/server/grpc_service.go

```diff
@@ -191,6 +185,48 @@ func (s *Service) StoreHeartbeat(ctx context.Context, request *schedulingpb.Stor
 	return &schedulingpb.StoreHeartbeatResponse{Header: &schedulingpb.ResponseHeader{ClusterId: s.clusterID}}, nil
 }
 
+// SplitRegions split regions by the given split keys
+func (s *Service) SplitRegions(ctx context.Context, request *schedulingpb.SplitRegionsRequest) (*schedulingpb.SplitRegionsResponse, error) {
```

Comment:
BTW, we seem to need to support `AskSplit`, `AskBatchSplit`, and `GetOperator` with grpc method too.

Reply:
We don't need AskSplit/AskBatchSplit. But GetOperator is necessary.

Reply:
`HandleAskBatchSplit` will call `GetMergeChecker()`

Reply:
I think we can move the split cache out, WDYT?

Reply:
Oh, you are right.

Reply:
Will do it in the next PR